.gitignore

```diff
@@ -207,4 +207,7 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
-# End of https://www.toptal.com/developers/gitignore/api/node,windows,linux,visualstudiocode
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/node,windows,linux,visualstudiocode
+
+#Reports
+report
```

Comment:
EOF

Reply:
Fixed in commit add mochawesome reportfix